joomla/joomla-cms
# [4.3] Com_banners string [a11y]

### Summary of Changes
The language key used to provide the aria label on the banner edit form is incorrect. This PR corrects that

### Testing Instructions
in the admin view the source of the banner edit form and the client edit form

Note that if it is a new banner or an edit banner the string is slightly different

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/234826762-825a5566-ab46-4e1d-aad2-05930290bd87.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/234826642-502479b5-71e9-4eb8-a49f-c9d77d498a8d.png)



### Link to documentations
Please select:
